### PR TITLE
Update HREX IG to 1.0.0

### PR DESCRIPTION
**Pull request by sfc42**
Signed-off-by: Susan Crowell <sfc@us.ibm.com>

### CODE REVIEW

File: conformance/fhir-ig-davinci-hrex/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# HL7 FHIR� Da Vinci Health Record Exchange (HRex) 0.2.0 - STU R1 - 2nd ballot
+# HL7 FHIR� Da Vinci Health Record Exchange (HRex) 1.0.0 - STU R1
```
Comment (lmsurpre) [inline conformance/fhir-ig-davinci-hrex/README.md:1]: what char is this?  would have expected ® like before

Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/README.md:1]: Hmm...  Not sure what happened here -- I will fix.

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/020/package/.index.json [hunk 2]
```diff
@@ -6,21 +6,21 @@
 			"resourceType": "ImplementationGuide",
 			"id": "hl7.fhir.us.davinci-hrex",
 			"url": "http://hl7.org/fhir/us/davinci-hrex/ImplementationGuide/hl7.fhir.us.davinci-hrex",
-			"version": "0.2.0"
+			"version": "1.0.0"
```
Comment (lmsurpre) [inline conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/020/package/.index.json:9]: i think we should change the package path to hl7/fhir/us/davinci-hrex/100/ to match

File: conformance/fhir-ig-davinci-hrex/src/test/resources/JSON/020/Bundle-subscription-notification-with-data.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
...
..."> (<a href=\"CodeSystem-hrex-temp.html\">HRex Temporary Code System</a>#data-value)</span></td><td><a name=\"results\"> </a><blockquote><div style=\"display: inline-block; background-color: #d9e0e7; padding: 6px; margin: 4px; border: 1px solid #8da1b4; border-radius: 5px; line-height: 60%\"><p style=\"margin-bottom: 0px\">Resource \"results\" </p></div><p><b>type</b>: searchset</p><p><b>total</b>: 1</p><h3>Links</h3><table class=\"grid\"><tr><td>-</td><td><b>Relation</b></td><td><b>Url</b></td></tr><tr><td>*</td><td>self</td><td><a href=\"http://example.org/new-payer/fhir/Condition?patient=Patient/pat1&amp;clinical-status=active&amp;code:below=http%3A%2F%2Fsnomed.info%2Fsct%7C73211009\">http://example.org/new-payer/fhir/Condition?patient=Patient/pat1&amp;clinical-status=active&amp;code:below=http%3A%2F%2Fsnomed.info%2Fsct%7C73211009</a></td></tr></table><blockquote><p><b>entry</b></p><p><b>fullUrl</b>: <a href=\"http://local.fhir.org:960/r4/Condition/example\">http://local.fhir.org:960/r4/Condition/example</a></p><h3>Searches</h3><table class=\"grid\"><tr><td>-</td><td><b>Mode</b></td></tr><tr><td>*</td><td>match</td></tr></table></blockquote></blockquote></td></tr></table></div>"
```
Comment (lmsurpre) [inline conformance/fhir-ig-davinci-hrex/src/test/resources/JSON/020/Bundle-subscription-notification-with-data.json:54]: i'd like these paths updated from JSON/020 to JSON/100 as well

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/020/package/.index.json (hunk 2 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/020/package/.index.json:9]: done

File: conformance/fhir-ig-davinci-hrex/src/test/resources/JSON/020/Bundle-subscription-notification-with-data.json (hunk 3 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/src/test/resources/JSON/020/Bundle-subscription-notification-with-data.json:54]: done

**Review by lmsurpre — APPROVED** [review]
LGTM

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/ImplementationGuide-hl7.fhir.us.davinci-hrex.json [hunk 4]
```diff
@@ -58,7 +58,7 @@
 			"id": "uscore",
 			"uri": "http://hl7.org/fhir/us/core/ImplementationGuide/hl7.fhir.us.core",
 			"packageId": "hl7.fhir.us.core",
-			"version": "4.0.0"
+			"version": "3.1.1"
```
Comment (prb112) [inline conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/ImplementationGuide-hl7.fhir.us.davinci-hrex.json:61]: Discussed as a team... it should be 3.1.1 consistently through this PR.

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json [hunk 5]
```diff
@@ -942,11 +942,11 @@
...
 						"targetProfile": [
-							"http://hl7.org/fhir/StructureDefinition/Organization",
-							"http://hl7.org/fhir/StructureDefinition/Patient",
-							"http://hl7.org/fhir/StructureDefinition/Practitioner",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient|4.0.0",
 							"http://hl7.org/fhir/StructureDefinition/RelatedPerson",
-							"http://hl7.org/fhir/StructureDefinition/PractitionerRole"
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner|4.0.0",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitionerrole|4.0.0",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-organization|4.0.0"
```
Comment (prb112) [inline conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json:949]: Please switch to 3.1.1
Also update the CHANGELOG

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json [hunk 6]
```diff
@@ -3417,6 +3680,25 @@
...
+				"max": "1",
+				"type": [
+					{
+						"code": "Reference",
+						"targetProfile": [
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient|4.0.0",
+							"http://hl7.org/fhir/StructureDefinition/RelatedPerson",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner|4.0.0",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitionerrole|4.0.0",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-organization|4.0.0"
```
Comment (prb112) [inline conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json:3696]: 3.1.1?

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-practitionerrole.json [hunk 7]
```diff
@@ -794,7 +806,8 @@
 				"isSummary": true,
 				"binding": {
 					"strength": "extensible",
-					"valueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.114222.4.11.1066"
+					"description": "Provider specialty codes consist of NUCC Health Care Provider Taxonomy Code Set for providers.",
+					"valueSet": "http://hl7.org/fhir/us/core/ValueSet/us-core-provider-specialty|4.0.0"
```
Comment (prb112) [inline conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-practitionerrole.json:810]: 3.1.1

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-provenance.json [hunk 8]
```diff
@@ -1543,46 +1569,8 @@
 						"code": "Reference",
 						"targetProfile": [
 							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner|4.0.0",
-							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-organization|4.0.0",
 							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient|4.0.0",
-							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitionerrole|4.0.0",
```
Comment (prb112) [inline conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-provenance.json:1548]: 3.1.1

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/ImplementationGuide-hl7.fhir.us.davinci-hrex.json (hunk 4 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/ImplementationGuide-hl7.fhir.us.davinci-hrex.json:61]: done

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json (hunk 5 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json:949]: done

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json (hunk 6 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json:3696]: done

File: conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-provenance.json (hunk 8 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-provenance.json:1548]: done

**Pull request merged**